server/docker.go

```diff
@@ -41,6 +41,12 @@ func (w *dockerWatcherImpl) makeWakerFunc(_ *routableContainer) func(ctx context
 	}
 }
 
+func (w *dockerWatcherImpl) makeSleeperFunc(_ *routableContainer) func(ctx context.Context) error {
+	return func(ctx context.Context) error {
+		return nil
```

Comment:
I'm wondering if both makeWakerFunc and this should be logging a fatal that auto up/down is not supported (yet) for Docker.